test/e2e/data/infrastructure-docker/v1alpha4/bases/cluster-with-kcp.yaml

```diff
@@ -71,4 +71,4 @@ spec:
       nodeRegistration:
         criSocket: /var/run/containerd/containerd.sock
         kubeletExtraArgs: {eviction-hard: 'nodefs.available<0%,nodefs.inodesFree<0%,imagefs.available<0%'}
-  version: "${KUBERNETES_VERSION}"
+  version: "${KUBERNETES_VERSION}"
```

Comment:
nit: GitHub complain if there's no new line at the end of file; also some editors, add this by default, so If possible leave it (applies to other yaml files as well)